src/community/wmts-multi-dimensional/src/main/java/org/geoserver/gwc/wmts/MultiDimensionalExtension.java

```diff
@@ -65,15 +69,25 @@
     public static final Set<String> ALL_DOMAINS = Collections.emptySet();
     /** Configuration key for the maximum allowed value of expand_limit */
     public static final String EXPAND_LIMIT_MAX_KEY = "MD_DOMAIN_EXPAND_LIMIT_MAX";
+
+    public static final String LIMIT_MAX_KEY = "MD_GET_DOMAIN_VALUES_LIMIT_MAX";
```

Comment:
Since there is already other ``LIMIT`` constants I would probably put something more meaningful than ``LIMIT* ``, e.g. ``DOMAIN_VALUES_LIMIT_MAX_KEY``.

Reply:
Fixing